setup.cfg

```diff
@@ -1,6 +1,3 @@
-[bdist_wheel]
-universal = 1
```

Comment:
This package is definitely universal FYI, as it supports py3 only.

Reply:
@gaborbernat I believe you meant "it is universal by mistake which should be changed" as we discussed today. Right? I did some testing with pip I'll make the suggested change. I've also documented this at https://github.com/pypa/packaging.python.org/issues/726#issuecomment-792815388.